sw/blas/gemm/src/gemm.h

```diff
@@ -73,8 +237,21 @@ void gemm_fp64_opt(uint32_t M, uint32_t N, uint32_t K, double* A, uint32_t ldA,
     // Unrolling factor of most inner loop.
     // Should be at least as high as the FMA delay
     // for maximum utilization
-    const uint32_t unroll = 8;
-
+    // const uint32_t unroll = 8;
+    const uint32_t unroll = 4;
```

Comment:
What's the reason to decrease the unrolling? I tried it myself once, since the FPU latency should now be low enough to prevent RAW conflicts, but I remember that it reduced the FPU utilization by a couple of percent, which is why I decided to keep it at 8.